packages/grid/x-grid/package.json

```diff
@@ -24,6 +24,12 @@
     "datagrid",
     "data-grid"
   ],
+  "scripts": {
+    "precommit": "npm run lint",
+    "lint": "../../../node_modules/.bin/tsc --noEmit && eslint 'src/**/*.{ts,tsx}' --quiet --fix -c ../../../.eslintrc.js && npm run lint:css",
```

Comment:
Not sure it's still useful the lint script here, if we need to lint from the root folder

Reply:
No strong preferences. We lint eslint from the root but typescript from each package. I personally never run eslint in my local env. I run TypeScript from time to time. Happy to only have eslint at the root.